src/Raven.Pal/Raven.Pal.vcxproj

```diff
@@ -46,7 +46,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <UseDebugLibraries>false</UseDebugLibraries>
+    <UseDebugLibraries>true</UseDebugLibraries>
```

Comment:
should be reverted?

Reply:
No, that isn't actually being used except for debugging purposes.